about.html

```diff
@@ -11,54 +11,67 @@
 
 </head>
 
+<header>
+    <div class="about-page">
+    <section Class="nav-bar">
```

Comment:
This is where you start your nav bar. Awesome! But you start it with a <section> tag. What do you end it with? Where is the end of this </section>?

I think it's on line 24, but you have </nav> instead of </section>.